Dockerfile

```diff
@@ -12,8 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch-slim
+FROM golang:1.12-stretch AS build
```

Comment:
any reason we don't want to use buster? Also could we upgrade to 1.13?

Reply:
yeah both of those updates should be fine